docs/TerraToolkitUpgradeGuide-v4.0.0.md

```diff
@@ -0,0 +1,101 @@
+# Terra Toolkit Upgrade Guide v4.0.0
+This document will provide information on upgrading from terra-toolkit 3.x to 4.0.0.
+
+## Terra Scripts
+The scripts provided by Terra Toolkit v3 used `tt:` prefixes. It was found `:` causes incorrect .bin generation on windows devices so these scripts have been updated to use the prefix `tt-`.
+```diff
+-    "aggregate-translations": "tt:aggregate-translations",
++    "aggregate-translations": "tt-aggregate-translations",
+-    "start": "tt:serve",
++    "start": "tt-serve",
+-    "start-static": "tt:serve-static",
++    "start-static": "tt-serve-static",
+```
+### Serve
+The `webpack-serve` dependency was updated from `^0.3.1` to `^1.0.2`.
+
+### Wdio Runner
+Terra Toolkit now offers its own wdio test runner which runs wdio test runs for specified locales and form factors. This allows for locale test runs that can match parallelized container test runs by concurrently running the wdio for each test run variation. This runner is a replacement for webdriver's bin script `wdio` by directly calling Webdriver's test launcher module for each test variation.
+
+Terra's wdio test runner is available via the `tt-wdio` cli or the `wdio-runner` javascript function.
+
+### Clean Screenshots
+Terra Toolkit now offers a screenshot cleanup tool to remove `errorScrenshots`, `latest`, `diff` and, if indicated, `reference` screenshots.
+
+This is available via the `tt-clean-screenshots` cli or the `clean-screenshots` javascript function.
+
+###### For more information about terra-wdio and clean-screenshots and for setup examples please go [here](https://github.com/cerner/terra-toolkit/tree/master/scripts/wdio).
+
+
+## Webpack Configuration
+### Configuration Changes
+The browsers list provided to the Autoprefixer plugin were updated to:
+- iOS >= 10
+- last 2 and_chr versions
+- last 2 android versions
+- last 2 chrome versions
+- last 2 edge versions
+- last 2 firefox versions
+- last 2 ie versions
+- last 2 safari versions'
+
+### Dependencies Updates
+The following dependencies were updated in Terra toolkit's default webpack configuration:
+
+- autoprefixer: `^6.7.7` -> `^8.5.2`
+- load-json-file: `^2.0.0` -> `^5.0.0`
+- postcss-assets-webpack-plugin: `^1.1.0` -> `^2.0.0`
+- postcss-custom-properties: `^6.0.1` -> `^7.0.0`
+- sass-loader: `^6.0.6` -> `^7.0.1`
+
+
+
+
+## WebdriverIO
```

Comment:
IMO I do not want to advertise this as a change. It would be better if projects followed the same package structure. However, if others feel this should be included I can add it.

Reply:
I can see either - there could potentially be consumers who had a need for custom specs, but couldn't with the last major release. Pretty indifferent though.